src/views/common/DashOverview.tsx

```diff
@@ -19,6 +27,28 @@ export const DashOverviewIntro: React.FC<{
   balance: BigNumber | undefined;
 }> = ({ asset, mode, onSubmit, amount, setAmount, balance }) => {
   const { data: decimals } = useDecimalCountForToken(asset.tokenAddress);
+  const { data: wNative } = useWrappedNativeDefinition();
+  const tokenAddress =
+    asset.tokenAddress === NATIVE_TOKEN
+      ? wNative?.tokenAddress
+      : asset.tokenAddress;
+
+  const newHealthFactor = useNewHealthFactorCalculator(
+    amount,
+    tokenAddress,
+    mode === "withdraw" || mode === "deposit" ? true : false,
+    mode === "deposit" || mode === "borrow" ? true : false
+  );
+
+  const maxAmount = useMaxChangeGivenHealthFactor(
+    balance,
+    tokenAddress,
+    mode,
+    BigNumber.from(1200)
```

Comment:
Make a module-level `export const MIN_SAFE_HEALTH_FACTOR = BigNumber.from(1200);` and use that constant here. `BigNumber.from(number)` is slow.